nightlies_watcher/hg_mozilla.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+
+def get_minimal_repository_name(repository):
+    return repository.split('/')[-1]
+
+
+def get_push_id(repository, revision):
+    response = requests.get(_get_push_log_url(repository, revision))
+    return _pluck_push_id(response.json(), revision)
+
+
+def _get_push_log_url(repository, revision):
+    return 'https://hg.mozilla.org/{}/json-pushes?changeset={}'.format(repository, revision)
+
+
+def _pluck_push_id(push_log_json, revision):
+    push_log_ids = tuple(push_log_json.keys())
```

Comment:
The tuple (or list) transformation is necessary 😞 . Otherwise this stack comes up when I return the first argument in the list:

```
Traceback (most recent call last):
  File "nightlies_watcher/watch.py", line 188, in <module>
    main(__name__)
  File "nightlies_watcher/watch.py", line 40, in main
    run_if_new_builds_are_present(config)
  File "nightlies_watcher/watch.py", line 65, in run_if_new_builds_are_present
    publish(config, revision, latest_task_definitions_per_achitecture)
  File "nightlies_watcher/watch.py", line 107, in publish
    hg_push_id = hg_mozilla.get_push_id(config['repository_to_watch'], revision)
  File "/home/jlorenzo/git/mozilla-releng/nightlies-watcher/nightlies_watcher/hg_mozilla.py", line 10, in get_push_id
    return _pluck_push_id(response.json(), revision)
  File "/home/jlorenzo/git/mozilla-releng/nightlies-watcher/nightlies_watcher/hg_mozilla.py", line 23, in _pluck_push_id
    return push_log_json.keys()[0]
TypeError: 'dict_keys' object does not support indexing
```